Aztec/Classes/TextKit/TextView.swift

```diff
@@ -141,7 +141,9 @@ open class TextView: UITextView {
         }
 
         string.loadLazyAttachments()
+
         storage.replaceCharacters(in: selectedRange, with: string)
+        selectedRange = NSRange(location: selectedRange.location + string.length, length: 0)
```

Comment:
I went back to double check, and the textViewDidChangeSelection is actually being called when you change the selecteRange, what is not being called is textViewDidChange.